components/fxa-client/src/oauth.rs

```diff
@@ -167,25 +167,52 @@ impl FirefoxAccount {
     /// Fetch an OAuth code for a particular client using a session token from the account state.
     /// This method doesn't support OAuth public clients at this time.
     ///
-    /// * `client_id` - OAuth client id.
```

Comment:
could we move this documentation either to `AuthorizationRequestParameters` or below under the `/// * `auth_params` Authentication parameters  ` , and possibly checking if they show up in Rustdoc?

Reply:
Alrighty done, They show up as a nested bulliet list in rus doc now. Also removed the bit about "not supporting public clients" since we added the PKCE params